device-types/Lenovo/SR850.yaml

```diff
@@ -0,0 +1,18 @@
+---
+manufacturer: Lenovo
+model: ThinkSystem SR850
+slug: sr850
+part_number: 7B27A03953
+u_height: 2
+is_full_depth: true
+comments: '[Lenovo System SR850 product guide](https://lenovopress.lenovo.com/lp1052.pdf)'
+subdevice_role: parent
```

Comment:
`subdevice_role: parent` should only be used when there are children devices, which would be housed under `device-bays`

An example of this is the Dell VRTX chassis. The chassis itself is a device, which is a parent to blades such as the M620.

Unless this device houses other devices, which I cannot find information saying it does, then please remove this definition.